cpp/benchmarks/ast/polynomials.cpp

```diff
@@ -80,7 +94,17 @@ static void BM_ast_polynomials(nvbench::state& state)
 
   state.exec(nvbench::exec_tag::sync, [&](nvbench::launch& launch) {
     cudf::scoped_range range{"benchmark_iteration"};
-    cudf::compute_column(*table, tree.back(), launch.get_stream().get_stream());
+
+    switch (engine) {
+      case engine_type::AST: {
+        cudf::compute_column(*table, tree.back(), launch.get_stream().get_stream());
+        break;
+      }
+      case engine_type::JIT: {
+        cudf::compute_column_jit(*table, tree.back(), launch.get_stream().get_stream());
+        break;
+      }
```

Comment:
This should cover all possible cases, otherwise the compiler my issue an error.
```suggestion
    switch (engine) {
      case engine_type::AST: {
        cudf::compute_column(*table, tree.back(), launch.get_stream().get_stream());
        break;
      }
      case engine_type::JIT: {
        cudf::compute_column_jit(*table, tree.back(), launch.get_stream().get_stream());
        break;
      }
      default: CUDF_FAIL("Invalid engine type");
```